README.rst

```diff
@@ -1839,51 +1839,51 @@ for configuring and running SQLAlchemy in a production system.
     from graphql_compiler.compiler.ir_lowering_sql.metadata import SqlMetadata
     from graphql_compiler.schema.schema_info import make_sqlalchemy_schema_info
     from graphql_compiler import graphql_to_sql
-    
+
```

Comment:
My editor deletes trailing spaces automatically. I don't think that we should not have trailing spaces in the README.rst, but I'd rather not make a pull request just to delete the trailing spaces so please allow this change in this PR.

Reply:
try using a better editor :)